fast64_internal/f3d/f3d_gbi.py

```diff
@@ -5344,110 +4378,47 @@ def gsDPLoadTileGeneric(c, tile, uls, ult, lrs, lrt):
     return words[0].to_bytes(4, "big") + words[1].to_bytes(4, "big")
 
 
-class DPSetTileSize:
-    def __init__(self, t, uls, ult, lrs, lrt):
-        self.t = t
-        self.uls = uls
-        self.ult = ult
-        self.lrs = lrs
-        self.lrt = lrt
+@dataclass
+class DPSetTileSize(GbiMacro):
+    t: int
+    uls: int
+    ult: int
+    lrs: int
+    lrt: int
 
     def to_binary(self, f3d, segments):
         return gsDPLoadTileGeneric(f3d.G_SETTILESIZE, self.t, self.uls, self.ult, self.lrs, self.lrt)
 
-    def to_c(self, static=True):
-        header = "gsDPSetTileSize(" if static else "gDPSetTileSize(glistp++, "
-        return (
-            header
-            + str(self.t)
-            + ", "
-            + str(self.uls)
-            + ", "
-            + str(self.ult)
-            + ", "
-            + str(self.lrs)
-            + ", "
-            + str(self.lrt)
-            + ")"
-        )
-
-    def to_sm64_decomp_s(self):
-        return (
-            "gsDPSetTileSize "
-            + str(self.t)
-            + ", "
-            + str(self.uls)
-            + ", "
-            + str(self.ult)
-            + ", "
-            + str(self.lrs)
-            + ", "
-            + str(self.lrt)
-        )
-
-    def size(self, f3d):
-        return GFX_SIZE
+    def is_LOADTILE(self):
+        return self.t == G_TX_LOADTILE
```

Comment:
I think `G_TX_LOADTILE` isn't globally defined like this?

Reply:
changed to f3d.G_TX_LOADTILE